lib/libzpool/kernel.c

```diff
@@ -481,9 +481,7 @@ cv_wait(kcondvar_t *cv, kmutex_t *mp)
 	ASSERT3U(cv->cv_magic, ==, CV_MAGIC);
 	ASSERT3P(mutex_owner(mp), ==, curthread);
 	mp->m_owner = MTX_INIT;
-	int ret = pthread_cond_wait(&cv->cv, &mp->m_lock);
-	if (ret != 0)
-		VERIFY3S(ret, ==, EINTR);
+	VERIFY3S(pthread_cond_wait(&cv->cv, &mp->m_lock), ==, 0);
```

Comment:
`VERIFY0(error)` is the same as `VERIFY3S(error, ==, 0)` and preferred for this kind of usage.